sparse/core.py

```diff
@@ -2412,19 +2412,19 @@ def maybe_densify(self, allowed_nnz=1000, allowed_fraction=0.25):
         >>> np.allclose(x, s.todense())
         True
 
-        You can also specify the minimum allowed sparsity or the maximum number
-        of nonzero values. If both conditions are unmet, this method will throw
+        You can also specify the minimum allowed density or the maximum number
+        of output elements. If both conditions are unmet, this method will throw
         an error.
 
         >>> x = np.zeros((5, 5), dtype=np.uint8)
         >>> x[2, 2] = 1
         >>> s = COO.from_numpy(x)
-        >>> s.maybe_densify(allowed_nnz=5, allowed_fraction=0.25)
+        >>> s.maybe_densify(max_size=5, min_fraction=0.25)
         Traceback (most recent call last):
             ...
         NotImplementedError: Operation would require converting large sparse array to dense
```

Comment:
You need to update this line in the docstring. [See here.](https://docs.python.org/3/library/doctest.html#what-about-exceptions)